data/repository/src/commonMain/kotlin/social/androiddev/common/repository/AuthenticationRepositoryImpl.kt

```diff
@@ -10,19 +10,26 @@
 package social.androiddev.common.repository
 
 import social.androiddev.common.network.MastodonApi
+import social.androiddev.common.persistence.localstorage.DodoLocalStorage
+import social.androiddev.domain.authentication.model.NewAppOAuthToken
 import social.androiddev.domain.authentication.repository.AuthenticationRepository
 
 internal class AuthenticationRepositoryImpl(
-    private val mastodonApi: MastodonApi
+    private val mastodonApi: MastodonApi,
+    private val localStorage: DodoLocalStorage,
 ) : AuthenticationRepository {
 
+    override suspend fun saveApplication(token: NewAppOAuthToken, domain: String) {
+        // TODO: Save metadata to Multiplatform Storage
```

Comment:
personally I like making issues/tickets and linking them to each todo for tracking but up to you

Reply:
All these todos are things I'll be doing in this PR. Just added them for draft PR review so nobody sees empty files or methods and thinks I forgot about it